easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.7.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author: Adam Huffman
+# adam.huffman@crick.ac.uk
+# The Francis Crick Institute
+# Based on work by npklein
+
+easyblock = 'PackedBinary'
+
+name = 'sratoolkit'
+version = '2.5.7'
+
+homepage = 'http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software'
+description = """The SRA Toolkit, and the source-code SRA System Development 
+ Kit (SDK), will allow you to programmatically access data housed within SRA
+ and convert it from the SRA format"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/%(version)s/']
+sources = ['sratoolkit.%(version)s-centos_linux64.tar.gz']
+
+executables = [
+    'abi-dump*', 'abi-load*', 'align-info*', 'bam-load*', 'blastn_vdb*', 'cache-mgr*', 
+    'cg-load*', 'fastq-dump*', 'fastq-load*', 'helicos-load*', 'illumina-dump*', 
+    'illumina-load*', 'kar*', 'kdbmeta*', 'latf-load*', 'ncbi', 'pacbio-load*', 
+    'prefetch*', 'rcexplain*', 'remote-fuser*', 'sam-dump*', 'sff-dump*', 
+    'sff-load*', 'sra-kar*', 'sra-pileup*', 'sra-sort*', 'sra-stat*', 'srapath*', 
+    'srf-load*', 'tblastn_vdb*', 'test-sra*', 'vdb-config*', 'vdb-copy*', 
+    'vdb-decrypt*', 'vdb-dump*', 'vdb-encrypt*', 'vdb-lock*', 'vdb-passwd*', 
+    'vdb-unlock*', 'vdb-validate*',
+]
+
+files_to_copy = [('*', 'bin')]
```

Comment:
@boegel - shall I just remove those two stanzas then?

Is that what's expected with `PackedBinary`?